target/scala-2.11/scala-space-launcher.js

```diff
@@ -0,0 +1,2 @@
+((typeof global === "object" && global &&
+         global["Object"] === Object) ? global : this)["scalaspace"]["ScalaSpace"]().main();
```

Comment:
You can simply change the directory in which they end up from the start with things like:
```scala
crossTarget in fastOptJS := baseDirectory.value / "js"
```
and similar for the other tasks that produce the various .js files.